### PR TITLE
use security printer

### PR DESCRIPTION
**Pull request by ildar-khisambeev**

### CODE REVIEW

File: ydb/core/tx/schemeshard/schemeshard__operation.cpp [hunk 1]
```diff
@@ -183,7 +191,7 @@ THolder<TProposeResponse> TSchemeShard::IgniteOperation(TProposeRequest& request
                                     << ", already accepted parts: " << operation->Parts.size()
                                     << ", propose result status: " << NKikimrScheme::EStatus_Name(response->Record.GetStatus())
                                     << ", with reason: " << response->Record.GetReason()
-                                    << ", tx message: " << GetRecordForPrint(record).ShortDebugString());
```
Comment (alexnick88) [inline ydb/core/tx/schemeshard/schemeshard__operation.cpp:186]: Old method masked UserToken. Do it masked with new method?

File: ydb/public/api/protos/ydb_persqueue_v1.proto [hunk 2]
```diff
@@ -38,7 +38,7 @@ message OffsetsRange {
 
 // In-session reauthentication and reauthorization, lets user increase session lifetime. You should wait for 'update_token_response' before sending next 'update_token_request'.
 message UpdateTokenRequest {
-    string token = 1;
+    string token = 1 [(Ydb.sensitive) = true];
```
Comment (alexnick88) [inline ydb/public/api/protos/ydb_persqueue_v1.proto:41]: Fix logging of this in GrpcProxy

File: ydb/core/tx/schemeshard/schemeshard__operation.cpp (hunk 1 above)
Comment (ildar-khisambeev) [reply ydb/core/tx/schemeshard/schemeshard__operation.cpp:186]: Yes

File: ydb/public/api/protos/ydb_persqueue_v1.proto (hunk 2 above)
Comment (ildar-khisambeev) [reply ydb/public/api/protos/ydb_persqueue_v1.proto:42]: Improved NYdbGrpc::FormatMessage using the TSecurityTextFormatPrinter

**Review by alexnick88 — APPROVED** [review]

**Pull request merged**